tests/components/lcn/test_scene.py

```diff
@@ -0,0 +1,72 @@
+"""Test for the LCN scene platform."""
+
+from unittest.mock import patch
+
+from pypck.lcn_defs import OutputPort, RelayPort
+
+from homeassistant.components.scene import DOMAIN as DOMAIN_SCENE
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_ON,
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .conftest import MockModuleConnection
+
+
+async def test_setup_lcn_scene(hass: HomeAssistant, lcn_connection) -> None:
+    """Test the setup of switch."""
+    for entity_id in (
+        "scene.romantic",
+        "scene.romantic_transition",
+    ):
+        state = hass.states.get(entity_id)
+        assert state is not None
+        assert state.state == STATE_UNKNOWN
+
+
+async def test_entity_attributes(
+    hass: HomeAssistant, entity_registry: er.EntityRegistry, entry, lcn_connection
+) -> None:
+    """Test the attributes of an entity."""
+    entity = entity_registry.async_get("scene.romantic")
+
+    assert entity
+    assert entity.unique_id == f"{entry.entry_id}-m000007-0.0"
+    assert entity.original_name == "Romantic"
+
+    entity_transition = entity_registry.async_get("scene.romantic_transition")
+
+    assert entity_transition
+    assert entity_transition.unique_id == f"{entry.entry_id}-m000007-0.1"
+    assert entity_transition.original_name == "Romantic Transition"
+
+
+@patch.object(MockModuleConnection, "activate_scene")
+async def test_scene_activate(
+    activate_scene, hass: HomeAssistant, lcn_connection
```

Comment:
Please have `hass` as the first parameter

Reply:
Afaik, the `patch` decorater will pass the mock always as first parameter when used with additional fixtures.
Do you want me to use a context manager for patching instead or leave it as it is?

Reply:
is there any other way of mocking that we can use?

Reply:
I guess the easiest thing besides passing the mock as first argument is wrapping the `hass.services.async_call` in a context manager. Just one line of additional code:
```python
with patch.object(MockModuleConnection) as activate_scene:
    await hass.services.async_call( ...

activate_scene.assert_waited_with( ...
```

Reply:
Sounds better imo